### DISCUSSION TITLE
Build out an example for customer accounts

### DISCUSSION THREAD

**Discussion by jackabox** [statamic-rad-pack/shopify#11] [Feature Requests]
You might want to replicate the customer accounts onto your site for things such as:

- Viewing Orders
- Seeing Updates
- Promos?

Would be good to scaffold out in the add-on resources and starter kit to give everyone a running jump with integrating account features.

**Comment by dougalcrowder** [comment]
I think this would be really useful.

**Comment by austriker27** [comment]
I would definitely love to see this!

**Comment by ryanmitchell** [comment]
Working on some tags for that over here https://github.com/statamic-rad-pack/shopify/pull/185
